tutorial/sample-problems/ProblemTechniques/RestrictAnswerToFraction.pg

```diff
@@ -13,7 +13,7 @@
 
 #:% name = Restrict Answers to a Fraction
 #:% type = [technique, sample]
-#:% subject = [answer]
+#:% subject = [algebra, precalculus]
```

Comment:
Add the `answers` category to this problem instead of just removing the `answer` subject.